StablecoinResearch/Stablecoins/Resources/ResearchPapers/papers_markdown/ADynamical Systems Approach to Cryptocurrency.md

```
@@ -0,0 +1,1268 @@
+
+A Dynamical Systems Approach to Cryptocurrency
+Stability
+
+Carey Caginalp,1,2∗
+
+1Department of Mathematics, University of Pittsburgh, USA
+301 Thackeray Hall, Pittsburgh PA 15260, USA
+2Economic Science Institute, Chapman University
+One University Drive, Orange CA 92866, USA
+
+∗To whom correspondence should be addressed; E-mail: carey caginalp@alumni.brown.edu.
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)